webapp/lib/charts.dart

```diff
@@ -162,3 +162,45 @@ class DailyTimeseriesLineChartView {
     chart.update(new chartjs.ChartUpdateProps(duration: 0));
   }
 }
+
+ class SystenEventsTimeseriesLineChartView extends DailyTimeseriesLineChartView {
```

Comment:
typo: `Systen` -> `System`

Reply:
Oops....I'll remember to rename it in the next branch

Reply:
Great, thanks, or you can just open a separate PR for it
(and sorry, this is my fault, I jumped the gun on merging this PR, I should have given you the chance to fix the typo here!)